### DISCUSSION TITLE
蓝灯无法正常登陆

### DISCUSSION THREAD

**Discussion by Janslo1990** [getlantern/lantern-forum-cn#938] [Q&A]
上次虽然给了我专业码，但是我尝试用邮箱登陆的时候，验证码一直没有发到邮箱来，很多次尝试登陆，一直等不到验证码，这有办法解决吗？

**Comment by Janslo1990** [comment]
蓝灯默认是没有登陆的吧？那么需要正常怎么登陆专业版？

**Reply by MaxMKMax** [reply]
@Janslo1990 您好. 请您提供我您的账号邮箱和支付单据. 我帮您让他们手动处理.

**Comment by Janslo1990** [comment]
邮箱是632979426@qq.com
希望尽快处理，我只是希望能正常使用，而不是连用都用不成

**Comment by Janslo1990** [comment]
![微信图片_20241023122240](https://github.com/user-attachments/assets/88b7c082-7701-42f2-8fb7-2a6c68a34a54)

**Comment by Janslo1990** [comment]
然后把专业版登陆流程告诉我，我验证码都收不到实在很难受

**Comment by MaxMKMax** [comment]
@Janslo1990 您好. 好的, 已经帮您提交上去解决问题了.

**Comment by greySkiesMarsh** [comment]
Hi @Janslo1990 

Please redeem this code to activate your account. V2BQT-JWFVH-3P3GJ-YF6MH-GYW9Q
